### DISCUSSION TITLE
Trouble creating endpoint in Nuxt 3 server with Firebase integration

### DISCUSSION THREAD

**Discussion by matifanger** [vuejs/vuefire#1337] [Questions and Help]
Hello everyone,

I'm currently facing some issues creating an endpoint in my Nuxt 3 app (/server/api). I'm not able to make VueFire work properly. 

useFirebase() seems to not be available there.

Has anyone faced a similar issue? Is there anyone who has successfully integrated VueFire with Nuxt 3 and created an endpoint? I'd appreciate any help or advice on this matter.

Thank you.

**Comment by dosstx** [comment]
No problems creating endpoint with regular Firebase NodeJS SDK. I am not using vuefire for that. Why would someone need VueFire for that?

**Reply by matifanger** [reply]
When VueFire is set up, it calls initializeApp. I would like to use the same instance, not another one. Is that even possible?

**Comment by chiuwah** [comment]
VueFire is only available in the Vue part of a Nuxt app. Since there is no Vue available in server api routes, you should use the Firebase Admin (NodeJS) SDK.
